docs/specification/appendix.md

```diff
@@ -53,6 +57,12 @@ Schema `encoding` may only be omitted for self-describing message encodings such
 - `encoding`: `protobuf`
 - `data`: A binary [FileDescriptorSet](https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/descriptor.proto) as produced by `protoc --descriptor_set_out`.
 
+### flatbuffer
+
+- `name`: FIXME
+- `encoding`: `flatbuffer`
+- `data`: FIXME
```

Comment:
A `reflection.Schema` flatbuffer describing the type of the topic, encoded as a binary flatbuffer. This is the schema that can be generated using `flatc -b --schema` run on the fbs file corresponding to the flatbuffer type in question.

TODO: Currently the code I have in https://github.com/foxglove/studio/pull/3465/files is written in such a way that the type of the topic must be the "root type" of the `reflection.Schema` object. This is a somewhat arbitrary  limitation, however, so long as the `name` is populated correctly.